test/Microsoft.AspNetCore.Cors.Test/CorsServiceTests.cs

```diff
@@ -881,9 +881,44 @@ public void ApplyResult_PreflightMaxAge_MaxAgeHeaderAdded()
             Assert.Equal("30", httpContext.Response.Headers["Access-Control-Max-Age"]);
         }
 
+        [Fact]
+        public void EvaluatePolicy_MultiOriginsPolicy_ReturnsVaryByOriginHeader()
+        {
+            // Arrange
+            var corsService = new CorsService(new TestCorsOptions());
+            var requestContext = GetHttpContext(origin: "http://example.com");
+            var policy = new CorsPolicy();
+            policy.Origins.Add("http://example.com");
+            policy.Origins.Add("http://example-two.com");
+
+            // Act
+            var result = corsService.EvaluatePolicy(requestContext, policy);
+
+            // Assert
+            Assert.NotNull(result.AllowedOrigin);
+            Assert.True(result.VaryByOrigin);
+        }
+
+		[Fact]
+		public void EvaluatePolicy_MultiOriginsPolicy_NoMatchingOrigin_ReturnsInvalidResult()
+		{
+			// Arrange
+			var corsService = new CorsService(new TestCorsOptions());
```

Comment:
Looks like the latest commit inserted tabs instead of spaces - can you change it back to all spaces?